packages/control-property-editor-common/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
Please rename this file to `.eslintrc.js`. This is what we use in all other packages and it is needed to avoid an eslint issue https://github.com/SAP/open-ux-tools/pull/1201

Reply:
renamed.

Reply:
Thx please see my other comment :D